### PR TITLE
[OCRVS-12095] Upgrade lodash to `^4.18.1`

### LINKED ISSUE #12095: We need to upgrade lodash and all dependencies that depend on lodash
### Description

This vulnerability wasnt picked up by Trivy yet, probably because the lodash version is newly released and Trivy database is not yet updated, meaning lodash needs to be upgraded to the latest version.

```
The identified library appears to be vulnerable.

https://login.XXXX .... assets/index-CYRTNoEY.js

="4.17.21",nt=200,ot="Unsupported core-js use. Try https://npms.io/search?q=ponyfill.",it="
Expected a function",at="Invalid `variable` option passed into `_.template`",st="
__lodash_hash_undefined__
```

Looking at yarn.lock there are a number of dependencies that refer to various lodash versions.

### Dev tasks

- [ ] Upgrade all dependencies that have lodash as a sub-dependency and ensure that versions used are patched with the fix for the above error

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
> [!NOTE]
> Currently, we do **not** run e2e tests as a check on `opencrvs-countryconfig`-repo PRs. Please ensure your PR doesn't break any e2e tests.
>
> One method for doing this is to open a PR with these changes to `opencrvs-farajaland` as well, and see if the PR check passes there.

## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,6 +128,7 @@
     ]
   },
   "resolutions": {
-    "acorn": "^6.4.1"
+    "acorn": "^6.4.1",
+    "lodash": "^4.18.1"
```
Comment (Zangetsu101) [inline package.json]: we probably don't need this tho

**Review by Zangetsu101 — APPROVED** [review]
